client/src/components/game/lab2/components/header/buttons/colorUpdate.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './../title.css';
+
+/*
+Component for the color update button on the color change popup
+*/
+const ColorUpdate = ({openColorChange}) => {
+  return (
+    <div>
+      <button
+        type="button"
+        className="updateColorButton"
+        onClick={openColorChange}
+      >
+        Update Colors
```

Comment:
I think this button needs to be centered maybe?
<img width="1148" alt="Screen Shot 2021-03-08 at 5 27 43 PM" src="https://user-images.githubusercontent.com/54957497/110390217-a2ec1380-8033-11eb-9d9f-4d394f738f43.png">

Reply:
It stays consistent with the previous header from the game page. I can center it, but I think it may look off.